wsbroadcastserver/clientmanager.go

```diff
@@ -372,25 +386,26 @@ func (cm *ClientManager) verifyClients() []*ClientConnection {
 	})
 
 	// metrics.NewSampleSnapshot doesn't require that its data is sorted, so it's fine that this is sorted in reverse order
-	clientsNonceScoreSample.SetSample(metrics.NewSampleSnapshot(int64(len(nonceScoreList)), nonceScoreList))
+	clientsNonceTargetSample.SetSample(metrics.NewSampleSnapshot(int64(len(nonceTargetList)), nonceTargetList))
 
 	for _, client := range recomputeNonceList {
 		now := time.Now()
-		newNonceHash := computeNonceHash(client.nonce, now)
+		newNonceHash := computeNonceHash(client.nonce, client.nonceTarget, now)
 		if client.nonceHash == newNonceHash {
 			client.nonceHashLastComputed = now
 			continue
 		}
-		_, exists := cm.clientPtrMap.Get(newNonceHash)
-		if exists {
-			// This nonce was already used on the next day
+		newNonceScore := scoreNonceHash(newNonceHash)
+		if newNonceScore < client.nonceTarget {
+			// This nonce no longer meets the target
 			clientDeleteList = append(clientDeleteList, client)
 			continue
 		}
```

Comment:
This will basically always trigger due to the target not being met for the new hash, right? Might as well just remove them unconditionally.

Reply:
It'll basically always trigger unless they have a zero threshold as clients that don't care about latency will. I figured this is a bit more generic anyways.

Reply:
Okay, that makes sense, it will stop clients that don't care from getting disconnected.